cmd/internal/cli/action_flags.go

```diff
@@ -830,6 +832,17 @@ var actionShareNSFlag = cmdline.Flag{
 	Hidden:       false,
 }
 
+// --execTimeout
```

Comment:
Change comment to `--runscript-timeout`

Reply:
good catch! Thanks for the comment, updated.